### ISSUE TITLE
Allow outputting OEM-DOS .fon files

### ISSUE THREAD

**Issue by yuiijjjjjjjjjjj** [fontforge/fontforge#4277]
They are .fon files, but a setting somewhere makes Microsoft Windows recognize the bitmap font as OEM-DOS. This makes it usable in command prompt environments.

**Comment by frank-trampe** [comment]
I don't understand exactly what you want, but it sounds like a small change and one that we are likely to accept. I'd suggest just making a pull request for this one.

**Comment by yuiijjjjjjjjjjj** [comment]
No, this would require additional interface to select the proper encoding.

**Comment by frank-trampe** [comment]
Right. You can just add a new preference checkbox for the flag or a new format that calls the TrueType exporter with the new flag. The interface is in `fontforgeexe/savefontdlg.c`, and the TrueType emission code is in `fontforge/tottf*.c`.

**Comment by yuiijjjjjjjjjjj** [comment]
But it isn't a boolean setting, it's a dropdown setting. I know this because in another bitmap font editor that has this setting you pick OEM with a dropdown.

**Comment by yuiijjjjjjjjjjj** [comment]
Also, .fon isn't a TrueType format!

**Comment by frank-trampe** [comment]
Oh, sorry about that. I thought that it was just another thing wrapped in TrueType. The inner code you want is in `FONFontDump` in `fontforge/winfonts.c`. Reading is in `SFReadWinFON`. If FON with the flag is better exposed as a different format in the drop-down box, just add another element in `savefontdlg.c` that calls the relevant internal routine with a flag to trigger the different export mode.
